maxfw/tests/test_image_utils.py

```diff
@@ -130,11 +130,24 @@ def test_imageprocessor_standardize():
     p = ImageProcessor(transform_sequence)
     img_out = p.apply_transforms(test_input)
     nose.tools.assert_almost_equal(np.std(img_out[..., :3]), 1)
-    nose.tools.assert_equal(np.std(img_out[..., 3]), 0)
+    nose.tools.assert_equal(np.mean(img_out[..., 3]), 0)
```

Comment:
Just checking, line above is using `:3` an here it is `3` - are we intending to check mean across all 3 channels, right?

Reply:
The `3` refers to the `A` in `RGBA`, which was all zeros. This check verifies if it's still the case, but I have added test cases for the three other (`RGB`) channels individually.

Reply:
In hindsight, I have removed this and am throwing an error for `RGBA` images instead, ref. other discussion below.